mycqu/score.py

```diff
@@ -13,16 +13,20 @@
 __all__ = ("Score", "GpaRanking")
 
 
-def get_score_raw(auth: Union[Session, str]):
+def get_score_raw(auth: Union[Session, str], isMinorBoo: bool):
     """
     获取学生原始成绩
+
     :param auth: 登陆后获取的authorization或者调用过mycqu.access_mycqu的session
     :type auth: Union[Session, str]
+    :param isMinorBoo: 是否获取辅修成绩
+    :type isMinorBoo: bool
     :return: 反序列化获取的score列表
     :rtype: Dict
     """
+    url = 'https://my.cqu.edu.cn/api/sam/score/student/score' + ('?isMinorBoo=true' if isMinorBoo else '')
```

Comment:
参数用 snake case 命名法，即用下划线分割单词。
此外这里建议给 `isMinorBoo` 一个默认值 `False` 以便向后兼容。

```suggestion
def get_score_raw(auth: Union[Session, str], is_minor_boo: bool):
    """
    获取学生原始成绩

    :param auth: 登陆后获取的authorization或者调用过mycqu.access_mycqu的session
    :type auth: Union[Session, str]
    :param is_minor_boo: 是否获取辅修成绩
    :type is_minor_boo: bool
    :return: 反序列化获取的score列表
    :rtype: Dict
    """
    url = 'https://my.cqu.edu.cn/api/sam/score/student/score' + ('?isMinorBoo=true' if is_minor_boo else '')
```

或

```suggestion
def get_score_raw(auth: Union[Session, str], is_minor_boo: bool = False):
    """
    获取学生原始成绩

    :param auth: 登陆后获取的authorization或者调用过mycqu.access_mycqu的session
    :type auth: Union[Session, str]
    :param is_minor_boo: 是否获取辅修成绩，默认为 :obj:`False`
    :type is_minor_boo: bool
    :return: 反序列化获取的score列表
    :rtype: Dict
    """
    url = 'https://my.cqu.edu.cn/api/sam/score/student/score' + ('?isMinorBoo=true' if is_minor_boo else '')
```